aead/Cargo.toml

```diff
@@ -12,3 +12,7 @@ categories = ["cryptography", "no-std"]
 
 [dependencies]
 generic-array = { version = "0.12", default-features = false }
+
+[features]
+default = ["alloc"]
```

Comment:
With AEAD crate implementations don't forget to write:
```toml
aead = { version = "0.1.2", default-features=false }
[features]
default = ["alloc"]
alloc = ["aead/alloc"]
```
And in the next minor version we should not forget to clean it up.